vmdb/app/controllers/application_controller.rb

```diff
@@ -573,6 +573,10 @@ def render_flash(add_flash_text = nil, severity = nil)
     end
   end
 
+  def tagging_explorer_controller?
```

Comment:
It's the best approach for what we have right now. :)

Since it's in a method we can more easily refactor later.  I think we'll be able to push this down, but for now I'm :+1: on it.